packages/kbn-i18n/README.md

```diff
@@ -273,8 +287,19 @@ Optionally we can pass `context` prop into `FormattedMessage` component.
 This prop is optional context comment that will be extracted by i18n tools
 and added as a comment next to translation message at `defaultMessages.json`
 
+In case when ReactJS component is converted to AngularJS directive using `reactDirective` AngularJS service, it's necessary to use React HOC `injectI18nProvider` to pass `intl` object to `FormattedMessage` component via context.
```

Comment:
__nit:__ ` is converted to AngularJS directive using` maybe instead `is rendered with the help of`?

Reply:
And maybe add one more line after example like this:

**NOTE:** To minimize the chance of having multiple `I18nProvider` components in the React tree, try to use `injectI18nProvider` or `I18nProvider` only to wrap the topmost component that you render, e.g. the one that's passed to `reactDirective` or `ReactDOM.render`.